fe/fe-core/src/main/java/com/starrocks/common/FeConstants.java

```diff
@@ -100,6 +100,9 @@ public class FeConstants {
 
     public static final int DEFAULT_UNPARTITIONED_TABLE_BUCKET_NUM = 16;
 
+    public static final String YES = "YES";
```

Comment:
These naming are too general. Maybe we can encapsulate them as:
public static class ShowSchemaConstants  {...}.

Or, will the YES and NO be used everywhere?

Reply:
done